osidb/models.py

```diff
@@ -588,10 +588,48 @@ def fts_search(q):
         # If search has no results, this will now return an empty queryset
 
 
+# Due to circular dependencies (on the Impact enum), this class must be
+# declared here
+# TODO: Move enums into their own namespace e.g. ./types.py
+class FlawMixin(ACLMixin, NullStrFieldsMixin):
+    """
+    Mixin for attributes in common between Flaw-like models
+    """
+
+    # short description of the Flaw
+    # TODO: should be a CharField?
+    title = models.TextField()
```

Comment:
This is the status quo, I suggest changing to a CharField because a TextField seems excessive, although I think postgres treats char columns with 255+ characters as text columns anyway so there wouldn't be a difference. I did not change it as it's an unnecessary migration IMO

Reply:
I am OK with the text field. Only stating what is the Bugzilla limit.